tedana/metrics/_utils.py

```diff
@@ -8,6 +8,29 @@
 LGR = logging.getLogger("GENERAL")
 
 
+def add_external_dependencies(dependency_config, external_regressor_config):
```

Comment:
Related to #704- can you add type annotations to all of the new or heavily modified functions?

Reply:
All functions in _utils.py now have type hints.